Mathlib/FieldTheory/Adjoin.lean

```diff
@@ -1634,6 +1634,19 @@ end IntermediateField
 
 end ExtendScalars
 
+section Finite
+
+namespace IntermediateField
+
+lemma finiteDimensional_of_le {E F : Type*} [Field F] [Field E] [Algebra F E]
+    {M N : IntermediateField F E} (le : M ≤ N) [FiniteDimensional F N] : FiniteDimensional F M :=
+  FiniteDimensional.of_injective (AlgHom.toLinearMap (IntermediateField.inclusion le))
+    (IntermediateField.inclusion_injective le)
```

Comment:
Maybe this should be `finiteDimensional_left_of_le` and there should be a `finiteDimensional_right_of_le` that says `FiniteDimensional M E` implies `finiteDimensional N E`?

Reply:
Matbe I shouldn't `make wheels` on my own from the start, this lemma is only a few instances away from `[FiniteDimensional.left](https://leanprover-community.github.io/mathlib4_docs/Mathlib/FieldTheory/Tower.html#FiniteDimensional.left)`